jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/WiperSetBuilder.java

```diff
@@ -41,34 +41,21 @@ public static J2735WiperSet genericWiperSet(JsonNode wiperSet) {
       J2735WiperSet gws = new J2735WiperSet();
 
       // statusFront and rateFront are required elements
-      int statusFront = wiperSet.get("statusFront").asInt();
-      int rateFront = wiperSet.get("rateFront").asInt();
-
-      if (statusFront < STATUS_LOWER_BOUND || STATUS_UPPER_BOUND < statusFront) {
```

Comment:
With removing this usage of the upper/lower bound status, we can also remove those static variables at the top of the file as they are no longer used.

Reply:
Oh nice

Reply:
Addressed